src/containers/BooksForm.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/no-array-index-key */
```

Comment:
- [ ] Please, do not disable eslint. They are meant to enforce best practices. Kindly learn how to fix it.